package.json

```diff
@@ -24,6 +26,8 @@
     "gulp-mocha": "~2.2.0",
     "gulp-mocha-phantomjs": "~0.11.0",
     "gulp-rename": "~1.2.2",
+    "gulp-sequence": "^0.4.5",
+    "gulp-template": "^3.1.0",
```

Comment:
gulp-template was added for setting version to bower.json file during a build. so, here I couldn't use gulp-header